website/server/controllers/api-v3/news.js

```diff
@@ -31,7 +31,6 @@ api.getNews = {
           <div class="mr-3 ${baileyClass}"></div>
           <div class="media-body">
             <h1 class="align-self-center">${res.t('newStuff')}</h1>
-            <h2>2/13/2020 - ${LAST_ANNOUNCEMENT_TITLE}</h2>
```

Comment:
is this HTML still used?

Reply:
Not really. I left this here for compatibility reasons of the API. But it should probably be changed to return the html rendered version of the latest post? Or should I just remove the API call?

Reply:
in theory we should have it return the html rendered version of the last post for compatibility but since it's used only internally afaik we might just skip this

Reply:
Actually it might not be too difficult to return the HTML version of the post? Not sure if it's worth it though, depends on whether any 3rd party tool uses it